systemd/systemd.go

```diff
@@ -437,12 +437,12 @@ func MountUnitPath(baseDir string) string {
 }
 
 func (s *systemd) WriteMountUnitFile(name, what, where, fstype string) (string, error) {
-	extra := ""
+	options := []string{"nodev"}
```

Comment:
we use devtmpfs on core which gets mounted by the initrd so the core snap itself never needs to manage /dev ... not sure if we might hit issues when the rootfs itself is mounted with nodev but i doubt it ... (we'll see in edge once this landed and i'll complain if all my boards stop working ;) ... i run everything on edge here)